docs/techreview.ltx

```diff
@@ -0,0 +1,440 @@
+
+\subsubsection{Selection}
+We will be using automatic scale selection.
+
+\subsection{Classification of AVIRIS data}
+\label{subsec:classification}
+
+\subsubsection{Options}
+\begin{enumerate}
+\item Conjugate-gradient backpropagation (CGBP)
+\item Gaussian maximum likelihood (GML)
+\item Minimum Euclidean distance (MED)
+\end{enumerate}
+\subsubsection{Goals}
+Before any further data analysis can take place, classifying the AVIRIS data into classes based on mineral concentrations and categories such as vegetation and bodies of water will be needed in order to have variables to correlate (or not) with the environmental effects of mining. 
+
+\subsubsection{Criteria}
+The main criteria being evaluated are accuracy based on tests performed in previous research and any aspects of the algorithms that could affect the accuracy during the training phase.
+
+\subsubsection{Comparison}
+\begin{tabular}{ | l | l | l | l |}
+    \hline
+    & CGBP & GML & MED \\ \hline
+    Test accuracy & 94.083\% & 90.48\% & 71.675\% \\ \hline
+    Confounding factors & $\times$  & \checkmark & $\times$ \\
+    \hline
+\end{tabular}
+\\ \\ This table shows an overview of CGBP, GML, and MED against the average overall test accuracy and presence of any confounding factors (yes or no) in a set of experiments conducted on AVIRIS data collected on Icelandic lands \cite{aviris}. GML is the only option that has a confounding factor. 
+
+\subsubsection{Discussion}
+In the experiments performed on AVIRIS data, there were no significant confounding factors that had an effect on the accuracy of the data when using the CGBP method. Just looking at the accuracy of the GML method, it looks like a fairly good choice, however this is a limitation to its use when using very high dimensional data like AVIRIS. The GML method relies on having "nonsingular (invertible)
+class-specific covariance matrices for all classes. [x]" Since there are often many dimensions to the data, there are cases when the sample size is smaller than the number of classes, which would make the data break the requirement of being nonsingular. The MED method performed the lowest in terms of accuracy and like the CGBP method, had no confounding factors that limited its usefulness. 
```

Comment:
This was supposed to cite the feature extraction and classification paper. I forgot to add the citation. I'll add it now.